end_to_end/signal_handler_test.go

```diff
@@ -61,7 +85,16 @@ var _ = Describe("Signal handler tests", func() {
 				"--copy-queue-size", "4",
 				"--verbose"}
 			cmd := exec.Command(gpbackupPath, args...)
+			outScanner, output := CreateOutput(cmd)
+
 			go func() {
+				// Wait for the first line on stdout
```

Comment:
Is it infinite scan? Is it possible to introduce timeout here?

Reply:
If the process freezes, then we wait on `cmd.Run()`. If the process ends without output, then `Scan` returns false.